src/containers/pages/InterventionPlan/IRS/plan/tests/index.test.tsx

```diff
@@ -117,4 +126,48 @@ describe('containers/pages/IRS/plan', () => {
 
     wrapper.unmount();
   });
+
+  it('calls the correct API endpoints to load map and table', () => {
+    store.dispatch(fetchPlanRecords([irsPlanRecordResponse1 as PlanRecordResponse])); // negates: opensrp (1) - get plan
+
+    const mock: any = jest.fn();
+    const { identifier: id } = irsPlanRecordResponse1;
+    // const supersetServiceMock: any = jest.fn();
+    // supersetServiceMock.mockImplementation(async () => jurisdictionsArray as FlexObject[]);
+
+    fetch
+      .mockResponseOnce(JSON.stringify([] as Organization[])) // opensrp (2) - get organizations: the only successful mock response
+      .mockResponseOnce(JSON.stringify(jurisdictionsArray as FlexObject[])) // superset (3) - get all jurisdictions
+      .mockResponseOnce(JSON.stringify([jurisdictionsArray[0]])); // opensrp (4) - get level 0 jurisdiction
```

Comment:
After dropping console logs in `componentDidMount` I can see that no combination of these works. In fact 

[According to the docs](https://www.npmjs.com/package/jest-fetch-mock#mocking-multiple-fetches-with-different-responses) we should be able to simply chain these responses for the multiple `fetch`s. However, I can see that the only `mockResponseOnce` being returned is `opensrp (2)` which is the first fetch being called in the test.